aea/helpers/storage.py

```diff
@@ -0,0 +1,432 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2020 Fetch.AI Limited
+#
```

Comment:
Suggest splitting: all apart from sql lite goes into `aea/helpers/storage/base.py` and then specific adaptors go into `aea/helpers/storage/sqlite.py` etc